### PR TITLE
[Core][MeshingApplication] Exporting interface SubModelpartListUtility to python

### PR DESCRIPTION
**Pull request by loumalouomega**
Plus removing evil -1, that wasn't used anyway (the test runs and everything looks OK, maybe I am wrong)

@miguelmaso looks like the origin of the error is not this, I PR this to avoid problems in the future with the -1

### CODE REVIEW

**Comment by miguelmaso** [comment]
I have just take a look to the code. The -1 was an useless and evil value

File: kratos/utilities/sub_model_parts_list_utility.h [hunk 1]
```diff
@@ -151,6 +151,11 @@ class KRATOS_API(KRATOS_CORE) SubModelPartsListUtility
         const std::string& SubModelPartName
         );
 
+    /**
+     * @brief This method can be used to debug complex model parts directly on python
+     */
+    void DebugComputeSubModelPartsList();
```
Comment (miguelmaso) [inline kratos/utilities/sub_model_parts_list_utility.h:157]: nice

**Review by miguelmaso — APPROVED** [review]

**Comment by miguelmaso** [comment]
is there an issue about the error?

**Comment by loumalouomega** [comment]
Nope, I just told you, hehe

**Pull request merged**